test/index.js

```diff
@@ -71,19 +72,27 @@ describe('Nvim', function() {
 
   it('can deal with custom types', function(done) {
     nvim.command('vsp', function(err, res) {
-      nvim.getWindows(function(err, windows) {
+      strictEqual(err, null);
+      nvim.listWins(function(err, windows) {
+        strictEqual(err, null);
         equal(windows.length, 2);
         equal(windows[0] instanceof nvim.Window, true);
         equal(windows[1] instanceof nvim.Window, true);
-        nvim.setCurrentWindow(windows[1], function(err, res) {
-          nvim.getCurrentWindow(function(err, win) {
+        nvim.setCurrentWin(windows[1], function(err, res) {
```

Comment:
Just as a sanity check, does this pass?

    nvim.setCurrentWin(0, function(err, res) {